tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "declaration": true,
     "strict": true,
     "noImplicitAny": true,
-    "outDir": "./lib"
+    "outDir": "./lib",
+    "skipLibCheck":true
```

Comment:
Is `puppeteer` causing any troubles with its type defs?

Reply:
Yes. It is drilling down to the type files in node_modules.